dask/dataframe/partitionquantiles.py

```diff
@@ -275,7 +277,11 @@ def merge_and_compress_summaries(vals_and_weights):
 
     Equal values will be combined, their weights summed together.
     """
-    vals_and_weights = [x for x in vals_and_weights if x]
+    vals_and_weights = [
+        (list(np.NaN if v is pd.NA else v for v in x[0]), x[1])
+        for x in vals_and_weights
+        if x
+    ]
```

Comment:
Do we always want to make this replacement? If so, what about moving this replacement logic closer to where `vals` are created?